lib/Select/Select.js

```diff
@@ -68,6 +68,19 @@ class Select extends Component {
     );
   }
 
+  getAriaLabelledBy() {
+    let ariaLabelledBy;
+    const label = this.props.label && `${this.id}-label`;
+    const readOnly = this.props.readOnly && `${this.id}-read-only-message`;
+    const ariaLabelledByString = [label, readOnly].filter(item => !!item).join(' ');
```

Comment:
Seems like we could just insert an external ariaLabelledBy prop here like so: 
```suggestion
    const ariaLabelledByString = [this.props.ariaLabelledBy, label, readOnly].filter(item => !!item).join(' ');
```

Reply:
Yeah, we could, but now we are hardcoding an order of references in the `aria-labelledby` string. I will merge it as is to include in stripes-components release and create another story to think over how to manage `aria-labelledby`.